Model/Cron.php

```diff
@@ -585,6 +641,12 @@ protected function _declareVariables($notification)
         $additionalData = !empty($notification->getAdditionalData()) ? $this->serializer->unserialize(
             $notification->getAdditionalData()
         ) : "";
+        if (!empty($additionalData[Vault::RECURRING_DETAIL_REFERENCE])) {
+            $this->_recurringDetailReference = $additionalData[Vault::RECURRING_DETAIL_REFERENCE];
+        }
+        if (!empty($additionalData[Vault::EXPIRY_DATE])) {
+            $this->_expiryDate = $additionalData[Vault::EXPIRY_DATE];
+        }
```

Comment:
I think it's not used anymore anywhere. Can it be removed?
```suggestion
```

Reply:
Indeed it can be removed in both locations